README.md

```diff
@@ -0,0 +1,4 @@
+This is the Wordpress theme for inclusivedesign.ca, the website of the Inclusive Design Institute.
```

Comment:
Another helpful note in the readme is, after "oAuth Twitter Feed for Developers plugin" is installed, follow [its installation doc](https://en-ca.wordpress.org/plugins/oauth-twitter-feed-for-developers/#installation) to provide the consumer key/secret etc in the admin settings page in order to get tweets displayed. This step is easy to be missed.